packages/@react-spectrum/datepicker/src/TimeField.tsx

```diff
@@ -52,6 +52,8 @@ function TimeField<T extends TimeValue>(props: SpectrumTimeFieldProps<T>, ref: F
 
   let validationState = state.validationState || (isInvalid ? 'invalid' : null);
 
+  let characterCount = useDateCharacterWidth(state) + 'ch';
```

Comment:
this isn't actually a character count though is it? nor a character width?

Reply:
Both... It's the character count being turned into a width. And on Picker it's a `calc()` which makes it more width.